src/sugar3/graphics/notebook.py

```diff
@@ -15,21 +15,25 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-"""Notebook class
+'''
+Notebook class
 
-This class create a Gtk.Notebook() widget supporting
-a close button in every tab when the 'can-close-tabs' gproperty
-is enabled (True)
+This class creates a :class:`Gtk.Notebook` widget supporting
+a close button in every tab when the `can-close-tabs` gproperty
+is enabled (True).
 
-STABLE.
-"""
+.. literalinclude ../examples/notebook.py
```

Comment:
For some reason this doesn't show up when sphinx renders the docs

[![https://gyazo.com/165e853a3ed7dd8572aa30385ba0f240](https://i.gyazo.com/165e853a3ed7dd8572aa30385ba0f240.png)](https://gyazo.com/165e853a3ed7dd8572aa30385ba0f240)